app/code/core/Mage/Tax/Helper/Data.php

```diff
@@ -1173,7 +1173,7 @@ public function getAllWeee($source = null)
      */
     public function isWrongDisplaySettingsIgnored()
     {
-        return (bool)$this->_app->getStore()->getConfig(Mage_Tax_Model_Config::XML_PATH_TAX_NOTIFICATION_PRICE_DISPLAY);
+        return $this->_isIgnored(Mage_Tax_Model_Config::XML_PATH_TAX_NOTIFICATION_PRICE_DISPLAY);
```

Comment:
The config is stored per store, but this PR will apply it to all stores. Is this OK?